content/en/news/releases/1.13.x/announcing-1.13/change-notes/index.md

```diff
@@ -0,0 +1,217 @@
+to 25 (from 100). This has been shown to improve performance under high load.
+
+
+- **Updated** the control plane to read `EndpointSlice` insead of `Endpoints`
+for service discovery for Kubernetes 1.21 or later. To switch back to the old
+`Endpoints` based behavior set `PILOT_USE_ENDPOINT_SLICE=false` in istiod.
+
+
+- **Added** support for hostname-based multi-network gateways for east-west traffic. The hostname will be resolved in
+the control plane and each of the IPs will be used as an endpoint. This behaviour can be disabled by setting
+`RESOLVE_HOSTNAME_GATEWAYS=false` for istiod.  ([Issue #29359](https://github.com/istio/istio/issues/29359))
+
+- **Added** support rewriting gRPC probes
+
+- **Added** a feature flag `PILOT_LEGACY_INGRESS_BEHAVIOR`, default to false.
+If this is set to true, istio ingress will perform the legacy behavior,
+which does not meet https://kubernetes.io/docs/concepts/services-networking/ingress/#multiple-matches.
+  ([Issue #35033](https://github.com/istio/istio/issues/35033))
+
+- **Fixed** listeners to balance between Envoy worker threads. Fixes (#18152)[https://github.com/istio/istio/issues/18152).  ([Issue #18152](https://github.com/istio/istio/issues/18152))
+
+- **Fixed** an issue where specifying conflict protocols for a service target port
+will cause unstable protocol selection for that port.
+  ([Issue #36462](https://github.com/istio/istio/issues/36462))
+
+- **Fixed** an issue where scaling endpoint for a service from 0 to 1
+might cause client side serivce account verification populated incorrectly.
+  ([Issue #36456](https://github.com/istio/istio/issues/36456))
+
+- **Fixed** an issue where TcpKeepalive setting at mesh config is not honored.
+  ([Issue #36499](https://github.com/istio/istio/issues/36499))
+
+- **Fixed** an issue where stale endpoints can be configured when a service gets deleted and created again.
+  ([Issue #36510](https://github.com/istio/istio/issues/36510))
+
+- **Fixed** an issue where istiod crashes if prioritized leader election (controlled via `PRIORITIZED_LEADER_ELECTION` env variable) is disabled.  ([Issue #36541](https://github.com/istio/istio/issues/36541))
+
+- **Fixed** an issue that sidecar iptables will cause intermittent connection reset due to the out of window packet.
+Introduced a flag meshConfig.defaultConfig.proxyMetadata.INVALID_DROP to control this setting.
+  ([Issue #36566](https://github.com/istio/istio/pull/36566))
+
+- **Fixed** an issue where in place upgrade will cause tcp connection between <1.12 proxy and 1.12 proxy to fail.
+  ([Issue #36797](https://github.com/istio/istio/pull/36797))
+
+- **Fixed** an issue where `EnvoyFilter` with ANY patch context will skip adding new clusters and listeners at gateway.
+
+
+- **Fixed** an issue causing HTTP/1.0 requests to be rejected (with a `426 Upgrade Required` error) in some cases.
+  ([Issue #36707](https://github.com/istio/istio/issues/36707))
+
```

Comment:
There are three empty lines between two "Fixed" items. Should the number of empty lines between items be consistent?

Reply:
I think this is just from the generation tool, but it renders OK:
https://deploy-preview-10849--preliminary-istio.netlify.app/latest/news/releases/1.13.x/announcing-1.13/change-notes/